### ISSUE TITLE
BUG : Un centre Dentaire ne s'affiche pas dans la catégorie dentiste et se téléporte aux USA lorsqu'on clique dessus

### ISSUE THREAD

**Issue by cayenne17** [cartesapp/cartes#668]
Bonjour,
Je suis tombé dessus par hasard sur ce bug en testant les différentes catégories.

Le problème concerne cet objet OSM : https://www.openstreetmap.org/relation/14141429

Sur cartes.app, il n'est pas mis en avant lorsque la catégorie "Dentiste" est activé : https://cartes.app/?cat=dentiste#17.21/46.389086/-0.418186/0/1

Lorsque je clique dessus, ça me téléporte aux États Unis d'Amérique et ça affiche un autre objet way (une route à la place) : https://www.openstreetmap.org/way/14141429.

Voici une courte vidéo du problème :

https://github.com/user-attachments/assets/bb16fd35-ed3e-4ddd-893d-8c0bc10b85dd

Caractéristique technique : 
- OS : Windows 10
- Navigateur : Vivaldi

**Comment by fouss** [comment]
Hello, je viens de jeter un œil, il semblerait que le problème proviennent directement d’OSM puisque les 2 objets ont le même ID !
![IMG_6060](https://github.com/user-attachments/assets/1cb059c6-0aec-4629-accc-37dbb3c0f7dd)
![IMG_6061](https://github.com/user-attachments/assets/4582312b-2524-4395-89e0-ee36771cccba)

**Comment by cayenne17** [comment]
> Hello, je viens de jeter un œil, il semblerait que le problème proviennent directement d’OSM puisque les 2 objets ont le même ID ! ![IMG_6060](https://private-user-images.githubusercontent.com/6073518/382230931-1cb059c6-0aec-4629-accc-37dbb3c0f7dd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA0NTA2ODksIm5iZiI6MTczMDQ1MDM4OSwicGF0aCI6Ii82MDczNTE4LzM4MjIzMDkzMS0xY2IwNTljNi0wYWVjLTQ2MjktYWNjYy0zN2RiYjNjMGY3ZGQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMDFUMDgzOTQ5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NDFmMmMyZGI0YTQxYTM1Nzk2MjJmOWY2ZmNjYzZkODEyNmI2M2RjOTNhODM0MDAzOWZjZDkzZWU1MThiYjNhNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.TLWfNXZRzygG9YK-hW9HoyIufAAeZyfMNJMr8cZiyQo) ![IMG_6061](https://private-user-images.githubusercontent.com/6073518/382230938-4582312b-2524-4395-89e0-ee36771cccba.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA0NTA2ODksIm5iZiI6MTczMDQ1MDM4OSwicGF0aCI6Ii82MDczNTE4LzM4MjIzMDkzOC00NTgyMzEyYi0yNTI0LTQzOTUtODllMC1lZTM2NzcxY2NjYmEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMDFUMDgzOTQ5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDg4Y2QzZmU1N2I4Y2IzYTJlZTdhYTk0NmEyNjUzMGRkZGVmYmFhMzM2YmZiN2U4MzdiNjcxYWFlZWRhNDgyNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.AA0C6LR7XAYoEOZNnu-cL9R3L6mW5NrY39WDL_htdjM)


Sauf qu'il y en a un c'est un type: way et l'autre un type: relation

**Comment by fouss** [comment]
Yes mais j’ai demande à chatGPT, ce n’est pas normal, c’est un problème d’OSM il faut faire un ticket 😬

**Comment by laem** [comment]
Voir "relations sans nwr" dans https://github.com/cartesapp/cartes/issues/651
Ça dépend de Tilemaker, je n'ai pas réussi à corriger le problème seul, et je n'ai pas de réponse de leur part.

**Comment by etienneJr** [comment]
Je cloture car les 2 soucis (présence dans catégorie dentiste, et clic téléporteur) sont résolus
